gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=28.3.7
+version=28.3.8
```

Comment:
There is a 28.3.8 already released. You might want to consider changing this?

Reply:
thanks, bumped again.